code/game/world.dm

```diff
@@ -48,6 +48,9 @@ GLOBAL_VAR(restart_counter)
 
 	GLOB.timezoneOffset = text2num(time2text(0,"hh")) * 36000
 
+	world.log << call("hookerino.dll","BHOOK_Init")()
```

Comment:
shouldn't this be uppercase H?

Reply:
on windows the case in paths doesn't matter